### PR TITLE
Component mockup

### PR DESCRIPTION
**Pull request by xueccc**
Create component with hardcoded data for mockup.

### CODE REVIEW

**Comment by xueccc** [comment]
@maspen or @libbyli Would you please review when you have a chance? Thank you!

File: client/src/components/productGrid.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ProductInfo from './productInfo.jsx'
+
+const ProductGrid = function(props) {
```
Comment (libbyli) [inline client/src/components/productGrid.jsx]: Try to use ES6

File: client/src/components/productInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const ProductInfo = function(props) {
```
Comment (libbyli) [inline client/src/components/productInfo.jsx]: Try to use ES6

File: client/src/index.jsx [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
+import 'bootstrap';
+import ShopInfo from './components/shopInfo.jsx';
+import SearchBar from './components/searchBar.jsx';
+import ProductGrid from './components/productGrid.jsx'
 import ShopInfo from './components/shopInfo.jsx'
```
Comment (libbyli) [inline client/src/index.jsx]: You're importing ShopInfo twice here, only need to import it once

**Comment by xueccc** [comment]
@libbyli I removed duplicate import and refactor function to use ES6. Would you please re-review?

**Pull request merged**